### PR TITLE
Doug/hw7

### PR DESCRIPTION
**Pull request by doughill1000**

### CODE REVIEW

**Comment by NavidG23** [comment]
Looks good to me :)

File: css/index.css [hunk 1]
```diff
@@ -112,8 +112,7 @@ footer {
 /*Patriot web title*/
 .greetingHeader {
     font-size: 75px;
-    /*font-family: URW Chancery L, cursive;*/
-    font-family: "Brush Script MT Bold", cursive;
+    font-family: "Brush Script MT Bold";
```
Comment (iTorfGMU) [inline css/index.css]: there needs to be a generic default for the font-family; also, the sub-header and the picture labels are still in the cursive script

File: css/index.css [hunk 2]
```diff
@@ -458,3 +461,12 @@ tr:nth-child(even) {
     font-weight: bold !important;
 }
 
+.registerSection h3{
+    font-weight: bold;
+    font-size:28px;
+}
+
+.navbar, .footer {
+    font-family: "Brush Script MT Bold";
```
Comment (iTorfGMU) [inline css/index.css]: see comment above

**Comment by doughill1000** [comment]
Ok changed the font to Arial Narrow for everything except tables and lists, which are arial I believe.

**Pull request merged**